Makefile

```diff
@@ -24,7 +24,7 @@ ${TOOLS}:
 	go get github.com/smartystreets/goconvey
 
 test: eremetic
-	go test -race -v ${PACKAGES}
+	go test -v ${PACKAGES}
```

Comment:
We could move this to ${TESTFLAGS} that defaults to "-v" and that way you could still do `make test TESTFLAGS="-v -race"` if you wanted to